setup.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from setuptools import (
+    setup,
+    find_packages,
+)
+
+extras_require = {
+    'test': [
+        "pytest==3.3.2",
+        "pytest-xdist",
+        "tox>=2.9.1,<3",
+    ],
+    'lint': [
+        "flake8==3.4.1",
+        "isort>=4.2.15,<5",
+        "mypy==0.701",
+        "pydocstyle>=3.0.0,<4",
+    ],
+    'doc': [
+        "Sphinx>=1.6.5,<2",
+        "sphinx_rtd_theme>=0.1.9",
+        "towncrier>=19.2.0, <20",
+    ],
+    'dev': [
+        "bumpversion>=0.5.3,<1",
+        "pytest-watch>=4.1.0,<5",
+        "wheel",
+        "twine",
+        "ipython",
+    ],
+}
+
+extras_require['dev'] = (
+    extras_require['dev'] +  # noqa: W504
+    extras_require['test'] +  # noqa: W504
+    extras_require['lint'] +  # noqa: W504
+    extras_require['doc']
+)
+
+
+with open('./README.md') as readme:
+    long_description = readme.read()
+
+
+setup(
+    name='dopple',
+    # *IMPORTANT*: Don't manually change the version here. Use `make bump`, as described in readme
+    version='0.4.0-alpha.0',
+    description="""dopple: HTTP proxy to Unix Socket based JSON-RPC servers""",
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    author='The Ethereum Foundation',
+    author_email='snakecharmers@ethereum.org',
+    url='https://github.com/ethereum/dopple',
+    include_package_data=True,
+    install_requires=[
+        "eth-utils>=1,<2",
```

Comment:
Why?

Reply:
Oh good catch, I guess this one is in the defaults of https://github.com/ethereum/ethereum-python-project-template because it is widely used but I agree that it should be removed in this case.